cirq/__init__.py

```diff
@@ -303,6 +303,10 @@
 from cirq.ion import (
     MS,
 )
+from cirq.neutral_atoms import (
+    AtomDevice,
```

Comment:
NeutralAtomDevice

Reply:
I've added neutral in all the places you've noted as well as the file names. Hopefully I caught them all.